docs/api/python/data_ingestion.md

```diff
@@ -95,112 +95,131 @@ con.execute('CREATE TABLE test_df_table AS SELECT * FROM test_df')
+
 ```python
 # example setting the sample size to 100000
 duckdb.default_connection.execute("SET GLOBAL pandas_analyze_sample=100000")
 ```
 
-### Object Conversion
+## Object Conversion
 
 This is a mapping of Python object types to DuckDB [Logical Types](../../sql/data_types/overview.html):
 
-- `None` -> `NULL`  
-- `bool` -> `BOOLEAN`  
-- `datetime.timedelta` -> `INTERVAL`  
-- `str` -> `VARCHAR`  
-- `bytearray` -> `BLOB`  
-- `memoryview` -> `BLOB`  
-- `decimal.Decimal` -> `DECIMAL` / `DOUBLE`  
-- `uuid.UUID` -> `UUID`  
-
-- #### `int`
-Since integers can be of arbitrary size in Python, there is not a one-to-one conversion possible for ints.  
-Intead we perform these casts in order until one succeeds:  
-  - `BIGINT`  
-  - `INTEGER`  
-  - `UBIGINT`  
-  - `UINTEGER`  
-  - `DOUBLE`  
-
-  When using the DuckDB Value class, it's possible to set a target type, which will influence the conversion.  
-
-- #### `float`
-These casts are tried in order:  
-  - `DOUBLE`  
-  - `FLOAT`  
-
-- #### `datetime.datetime`
-  For `datetime` we will check `pandas.isnull` if it's available and return `NULL` if it returns true.  
-  We also support `+inf` and `-inf` conversions.  
-
-  If the `datetime` has tzinfo, we will use `TIMESTAMPTZ`, otherwise it becomes `TIMESTAMP`.  
-
-- #### `datetime.time`
-  If the `time` has tzinfo, we will use `TIMETZ`, otherwise it becomes `TIME`.  
-
-- #### `datetime.date`
-  We support `+inf` and `-inf` conversions.  
-  `date` converts to the `DATE` type.  
-
-- #### `bytes`
-  `bytes` converts to `BLOB` by default, when it's used to construct a Value object of type `BITSTRING`, it maps to `BITSTRING` instead.  
-
-- #### `list`
-  `list` becomes a `LIST` type of the "most permissive" type of its children, for example:  
-```py
+* `None` -> `NULL`
+* `bool` -> `BOOLEAN`
+* `datetime.timedelta` -> `INTERVAL`
+* `str` -> `VARCHAR`
+* `bytearray` -> `BLOB`
+* `memoryview` -> `BLOB`
+* `decimal.Decimal` -> `DECIMAL` / `DOUBLE`
+* `uuid.UUID` -> `UUID`
+
+The rest of the conversion rules are as follows.
+
+### `int`
+
+Since integers can be of arbitrary size in Python, there is not a one-to-one conversion possible for ints.
+Intead we perform these casts in order until one succeeds:
+
+* `BIGINT`
+* `INTEGER`
+* `UBIGINT`
+* `UINTEGER`
+* `DOUBLE`
+
+When using the DuckDB Value class, it's possible to set a target type, which will influence the conversion.
+
+### `float`
+
+These casts are tried in order until one succeeds:
+
+* `DOUBLE`
+* `FLOAT`
+
+### `datetime.datetime`
+
+For `datetime` we will check `pandas.isnull` if it's available and return `NULL` if it returns true.
+We also support `+inf` and `-inf` conversions.
+
+If the `datetime` has tzinfo, we will use `TIMESTAMPTZ`, otherwise it becomes `TIMESTAMP`.
+
+### `datetime.time`
+
+If the `time` has tzinfo, we will use `TIMETZ`, otherwise it becomes `TIME`.
+
+### `datetime.date`
+
+We support `+inf` and `-inf` conversions.
```

Comment:
This might be a little confusing, and needs some context
What I mean by this is that we check for these values and treat them as `infinity` and `-infinity`:

```c++
bool PyDate::IsPositiveInfinity() const {
	return year == 9999 && month == 12 && day == 31;
}

bool PyDate::IsNegativeInfinity() const {
	return year == 1 && month == 1 && day == 1;
}
```

Reply:
These are taken from:
```py
>>> import datetime
>>> datetime.date.max 
datetime.date(9999, 12, 31)
>>> datetime.date.min 
datetime.date(1, 1, 1)
```

Reply:
`datetime` has the same treatment, but the values are of course slightly different:
```py
datetime.datetime(1, 1, 1, 0, 0)
>>> datetime.datetime.max
datetime.datetime(9999, 12, 31, 23, 59, 59, 999999)
```